plugins/modules/system/python_requirements_info.py

```diff
@@ -133,15 +134,24 @@ def main():
         match = pkg_dep_re.match(dep)
         if match is None:
             module.fail_json(msg="Failed to parse version requirement '{0}'. Must be formatted like 'ansible>2.6'".format(dep))
-        pkg, op, version = match.groups()
+        pkg, extra, op, version = match.groups()
         if op is not None and op not in operations:
```

Comment:
Not exactly related to the problem but, this check is redundant: if op is not one of the allowed ones, the regexp should fail and that would be caught by the previous check (`if match is None`).